src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -96,15 +97,15 @@
       <header id="resolution-date-header">
         <h2>{{getCreateResolutionResource.resolutionDateSection.header}}</h2>
       </header>
-      <p class="mt-2">
+      <p class="section-description mt-2">
         {{getCreateResolutionResource.resolutionDateSection.description}}
       </p>
       <div :class="{ 'invalid-section': getShowErrors && !this.isResolutionDateValid }">
         <v-card flat id="resolution-date-card" class="pt-7 pb-5 pl-6 pr-6">
           <v-row>
             <v-col id="resolution-date-card-left-col" cols="2" class="pt-6" >
               <v-card-title class="resolution-date-vcard-title pl-1 pr-0">
-                <div>Resolution Date</div>
```

Comment:
I agree; these divs are needed.

No need to change, but sometimes I wrap title text or button text in spans (eg, when they are on multiple lines) so that we can select the text directly in the unit tests and then use `toBe` instead of `toContain` (due to extra line breaks in the latter).

Reply:
^^ oops, typo. Added "not".